src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -466,13 +466,35 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
         return emails;
     }
 
+    /**
+     * Generates the feedback session closed email for both
+     * the instructors and the students with the given
+     * {@code userName} and {@code userEmail}.
+     */
     private EmailWrapper generateFeedbackSessionClosedEmail(CourseAttributes course,
             FeedbackSessionAttributes session, String userName, String userEmail) {
+
         String template = EmailTemplates.USER_FEEDBACK_SESSION_CLOSED;
         String subject = EmailType.FEEDBACK_CLOSED.getSubject();
+        String emailBody;
+        String instructorFragmentArgument;
+
+        /*Check if the given userName and userEmail correspond
+        to an Instructor or a Student and give the corresponding value
+        to the instructorFragmentArgument variable. */
+        if (instructorsLogic.isEmailOfInstructorOfCourse(userEmail, course.getId())) {
+            instructorFragmentArgument = "<p>The email below has been sent to students of course: "
```

Comment:
Could generate the fragment `instructorFragmentArgument` in the calling method on lines 453-467 and pass it in here as a parameter here to avoid making an datastore query for each generated email.